src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+body,
+li {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  height: 40px;
+}
+
+.nav__list {
+  height: 60px;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  text-align: center;
+  list-style: none;
+}
+
+.list__item {
+  position: relative;
+  border-bottom: #00acdc 4px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  color: #000;
+}
+
+.list__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: block;
```

Comment:
Why do you need to make it block?

Reply:
Because without block, my :after element cling to bottom of text "Apple"

Reply:
Doesn't it work?